examples/timebase_master.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python3
+
+"""JACK client that set timebase master.
+
+Usage : timebase_master.py [<BPM> [<beats per bar> [<ticks per beat>]]]
+
+This Client will register itself as `timebase master` and set extended
+position information :
+
+    * BPM
+    * Bar, Beat and Tick according to current position
+    * beat per bar, ticks per beat
+    * beat type, bar start tick
+
+"""
+import jack
+import sys
+
+argv = iter(sys.argv)
+next(argv)  # skip script name
+
+BPM = next(argv, 130)
+BAR_START_TICK = 0.0
+BEATS_PER_BAR = float(next(argv, 4.0))
+BEAT_TYPE = 4.0
+TICKS_PER_BEAT = float(next(argv, 960.0))
+
+client = jack.Client("Timebase Master")
+
+print("BPM: %s" % BPM)
+print("Beats per bar : %s" % BEATS_PER_BAR)
+print("Ticks per beat: %s" % TICKS_PER_BEAT)
+
+
+@client.set_timebase_callback
+def timebase_callback(state, blocksize, pos, new_pos):
+
+    if new_pos:
+        print("Position changed")
+
+    if not pos.frame_rate:
+        print("Frame position : %s" % pos.frame)
+        print("Frame rate : %s" % pos.frame_rate)
+        print("State : %s" % state)
+        print("Valid : %s" % pos.valid)
+
+    if pos.valid & 0x10:
+        check_modified_fields(pos)
```

Comment:
If any of the other "valid" flags are set, you should print a message, because we are ignoring this case.